sympy/functions/elementary/hyperbolic.py

```diff
@@ -209,6 +209,16 @@ class cosh(HyperbolicFunction):
     sinh, tanh, acosh
     """
 
+    @property
+    def is_positive(self):
+        if self.args[0].is_real:
+            return True
```

Comment:
Correction: That is not exactly wrong but there are other (non-real) argument values for which cosh is positive or negative. Perhaps this should include all of them?